widgetssdk/src/main/java/com/glia/widgets/view/floatingvisitorvideoview/FloatingVisitorVideoView.java

```diff
@@ -29,6 +29,12 @@ public FloatingVisitorVideoView(@NonNull Context context, @Nullable AttributeSet
         init();
     }
 
+    @Override
+    protected void onDetachedFromWindow() {
+        releaseVideoStream();
+        super.onDetachedFromWindow();
```

Comment:
does super call have to be the last in the method?

Reply:
re-arranged